client/babylon/game.jsx

```diff
@@ -231,6 +234,7 @@ function redrawTexture(mesh, newURL) {
 
 function addDrawingObservable(instance, scene, drawingMesh, socket) {
   instance.drawingObservable = scene.onBeforeRenderObservable.add(() => {
+    console.log('getting local canvas');
```

Comment:
please remove or comment out this console.log

Reply:
done